tasks/main.yml

```diff
@@ -5,13 +5,15 @@
     aws_access_key: "{{ lookup('env', 'AWS_ACCESS_KEY') }}"
     aws_secret_key: "{{ lookup('env', 'AWS_SECRET_KEY') }}"
     state: present
-    region: '{{ item.region }}'
-    vpc_id: '{{ item.vpc_id }}'
-    az: '{{ item.az }}'
-    cidr: '{{ item.cidr }}'
+    region: "{{ item.region }}"
+    vpc_id: "{{ item.vpc_id }}"
+    az: "{{ item.az }}"
+    cidr: "{{ item.cidr }}"
     resource_tags:
-      Name: '{{ item.Name }}'
-      Application: '{{ item.app_name }}'
-      Environment: '{{ item.app_environment }}'
-      ManagedByAnsible: '{{ ansible_vpc_subnets_tags_ManagedByAnsible }}'
-  with_items: '{{ subnets }}'
+      Name: "{{ item.Name }}"
+      Application: "{{ item.app_name }}"
+      Environment: "{{ item.app_environment }}"
+      app_environment: "{{ item.app_environment }}"
```

Comment:
Did you intend to add the `app_environment` tag, or did it just get missed from the commit/pr messages?